ci/lib/templates.lib.txt

```diff
@@ -203,6 +203,9 @@ git_rev=$(git rev-parse HEAD)
 popd
 version=$(cat version/number)
 builddate=$(date "+%Y-%m-%d")
+(@- if use_vcvarsall: @)
+remote_shell "c:/Program Files (x86)\Microsoft Visual Studio/2019/Community/VC/Auxiliary/Build/vcvarsall.bat" x64 10.0.16299.0 -vcvars_ver=14.20
```

Comment:
How are these env changes surviving past this single invocation of the command processor?

Needed my vcvars is a heavy hammer. A small fix to ACE CMake file should pass all you need to ACE.

Reply:
**Great catch. I**t worked on my local system because it's a single cmd shell. I took a look at how to do this directly in ACE's CMakeLists.txt, but it doesn't look like MPC has a value_template for the tools version. Any ideas on this would be greatly appreciated. Otherwise, can I just pass the vcvarsall.bat as the first argument to the remote_shell invocation that does the cmake config?